docs/content/en/docs-dev/user-guide/managing-application/configuration-drift-detection.md

```diff
@@ -53,3 +53,54 @@ This status means the application is deploying and the configuration drift detec
 This feature is automatically enabled for all applications.
 
 You can change the checking interval as well as [configure the notification](../../managing-piped/configuring-notifications/) for these events in `piped` configuration.
+
+### Enable to ignore drift detection for specific fields
+
+>  Note: This feature is currently supported for only kubernetes application.  
+
+This feature ignores drift detection for the specified fields. In other words, even if the specified fields are different between live state and Git, the application status will not be `Out of Sync`.
+
+To use this feature, set parameters in the application configuration. For more information, see the [configuration reference](../../configuration-reference/#driftdetection) and the following `Syntax` section.
+
+#### Syntax
+There are two kinds of block collection in yaml.
+
+Sequence: This block collection is indicated each entry with a dash and space ("`- `").  
+Mapping: This block collection is indicated each entry with a colon and space("`: `").
+
+Mapping indexes are specified, for example `.foo`.  
+Sequence indexes are specified, for example `.2`.  
+However, the first index doesn't need the `.`.
+
```

Comment:
I think we don't need this blog, users are expected to understand `yaml` syntax or if not we have to add this to every other places 👀

Reply:
I see. I will remove this section. 
Thank you:smile:

Reply:
Also, should move L63 under the example as well

Reply:
I fixed it 😃